patches/net/minecraft/client/resources/model/ModelResourceLocation.java.patch

```diff
@@ -0,0 +1,21 @@
+--- a/net/minecraft/client/resources/model/ModelResourceLocation.java
++++ b/net/minecraft/client/resources/model/ModelResourceLocation.java
+@@ -8,6 +_,7 @@
+ @OnlyIn(Dist.CLIENT)
+ public record ModelResourceLocation(ResourceLocation id, String variant) {
+     public static final String INVENTORY_VARIANT = "inventory";
++    public static final String NEOFORGE_SPECIAL_VARIANT = "neoforge_special";
+ 
+     public ModelResourceLocation(ResourceLocation id, String variant) {
+         variant = lowercaseVariant(variant);
+@@ -21,6 +_,10 @@
+ 
+     public static ModelResourceLocation inventory(ResourceLocation p_352141_) {
+         return new ModelResourceLocation(p_352141_, "inventory");
++    }
++
++    public static ModelResourceLocation neoforgeSpecial(ResourceLocation id) {
```

Comment:
Should be capitalized as `neoForgeSpecial`. It would be nice to include javadoc linking to the corresponding event I think.